chart/templates/mayastor/csi/csi-node-daemonset.yaml

```diff
@@ -23,8 +23,6 @@ spec:
       hostNetwork: true
       imagePullSecrets:
         {{- include "base_pull_secrets" . }}
-      nodeSelector:
-        kubernetes.io/arch: amd64
```

Comment:
Hi @xin3liang, we officially don't support any other architecture other than `amd64` and there is no upcoming plan of supporting any other architecture as now. So probably we don't need to remove this selector as anyhow there isn't any other architecture images. cc @GlennBullingham